types/backbone-relational/index.d.ts

```diff
@@ -1,15 +1,16 @@
-// Type definitions for Backbone-relational 0.8.5
+// Type definitions for Backbone-relational 0.10.0
 // Project: http://backbonerelational.org/
 // Definitions by: Eirik Hoem <https://github.com/eirikhm>
+//                 Julian Gonggrijp <https://github.com/jgonggrijp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.3
 
 /// <reference types="jquery" />
 
-import * as Backbone from 'backbone';
+import { EventsMixin, Events, Model as BModel, Collection } from 'backbone';
 
-declare module 'backbone' {
-    class RelationalModel extends Model {
+declare module 'backbone-relational' {
+    class Model extends BModel {
```

Comment:
A bit confused about the `RelationalModel` going away, as well as the module name change from `backbone` to `backbone-relational`. Were the old typings simply wrong? The new version does seem intuitively more correct but I'm trying to figure out what the reason was for the old definition.

Reply:
This change actually belongs with #32883, which has already been merged and released. The typings were outdated; backbone-relational used to monkey-patch the backbone module and insert `RelationalModel` into it, but nowadays it just exports that class as `Model` from its own module.

I mentioned the fact that this PR builds on top of #32883 in the description and suggested using a [custom diff](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32909/files/b737dc1ea56951ca7d06907c6a2433f0bebe6f36..3a37c7905905c5a5a6d991c4da6717f8f707db39) that excludes those commits for this reason. Perhaps I didn't state this clearly enough; sorry for that.

Reply:
Thanks, I saw your notice. But since #32883 was already merged I assumed the diff in this PR was now accurate. I'll wait for an update from you.